README.md

```diff
@@ -100,7 +100,13 @@ def start(_type, _args) do
 end
 ```
 
-Then you can use the pool as following:
+DBConnection defaults to [10 Poolboy connections](https://hexdocs.pm/db_connection/1.1.3/DBConnection.Poolboy.html#content), but you can change that with the `:pool_size` option:
```

Comment:
Can you change `DBConnection` to `DBConnection.Poolboy` to prevent people from thinking that this is the default behaviour? It should be clear that this is only when using the poolboy specific module.

Reply:
Oh, I didn't realize that, good clarification.